regression/heap/dll1_simple/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
These fail because we split abstract dynamic objects into multiple and such abstraction is not strong enough to reason about these programs. Should I create an issue for a stronger shape domain?